modules/op-using-secrets-and-workspaces.adoc

```diff
@@ -0,0 +1,367 @@
+// This module is included in the following assembly:
+//
+// *openshift-docs/cicd/pipelines/authenticating-pipelines-and-tasks-using-secrets.adoc
+
+[id="op-using-secrets-and-workspaces_{context}"]
+= Using secrets and workspaces
+
+Using Secrets and Workspaces in the Tekton pipeline provides a flexible approach to managing credentials in Tekton pipelines. You can bind a Secret to a Workspace, and then use that Secret content in your Task. 
+
+The following examples illustrate how to use Secrets and Workspaces.
+
+== git-clone Task
+
+This approach involves creating a task called `git-clone`, which clones a git repository using SSH authentication. The following are the steps to use Secrets and Workspace in Tekton Pipelines:
+
+. Create a Task called `git-clone` that clones a git repository using SSH authentication. 
+
+. Define workspaces, describe the process to create a secret, and bind it to the workspace. For example:
```

Comment:
1. I don't understand this part `describe the process to create a secret`
2. I split this point into two - one would only talk about how to create a secret, another one would talk about workspace declaration in task and binding.

Reply:
Noted, 
Resolved